docs/configuration.rst

```diff
@@ -82,6 +83,14 @@ Later steps in the documentation will change depending on your selection.
       For example, you might have several standard Crazyflies with arrangement 1,
       and several larger quadcopters with arrangement 2.
 
+   .. group-tab:: Single Marker
+
+      A special case of duplicated marker arrangements is the case where we only use a single marker per robot.
+      As before, the Crazyswarm will use the raw point cloud from the motion capture system and track the CFs frame-by-frame.
+      In this mode, we use optimal task assignment at every frame, which makes this mode more robust to motion capture outliers compared to the duplicate marker arrangements.
+      The main disadvantage is that the yaw angle cannot be observed without moving in the xy-plane.
```

Comment:
Would be nice if we could offer guidance on how long you can hover before yaw drift becomes significant.

Reply:
Done!